roles/splunk_common/tasks/extract_app.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Extract app to location
```

Comment:
Should this also be extended to support apps from a URL?

Reply:
Additionally, it would be good to list of apps this would not be compatible with. 

For example, ESS cannot be installed with just an `unarchive` - I think you need to install it through the REST API in order to pull down the TAs and other packages.

Reply:
For our use case we don't need URL get, but I can the fetch + extract tasks in this file. 

As for ES and ITSI, yes, these require different steps to install on a SHC which I believe is why the apps are installed locally to the deployer (or CM) then copied to the bundle dir and pushed.  I'll add a comment here mentioning this and add it to the docs as well.